manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
Maybe the katello module should install and configure the dependencies.

The pulp module should only take care of pulp (again, that's from my point of view).